cmd/sonobuoy/app/args.go

```diff
@@ -253,6 +254,14 @@ func AddRunWaitFlag(flag *int, flags *pflag.FlagSet) {
 	flags.Lookup("wait").NoOptDefVal = "1440"
 }
 
+// AddTimeoutFlag adds an int flag for waiting for the entire run to finish.
+func AddTimeoutFlag(flag *int, flags *pflag.FlagSet) {
+	flags.IntVar(
+		flag, timeoutFlag, config.DefaultAggregationServerTimeoutSeconds,
+		"How long (in seconds) Sonobuoy will wait for plugins to complete before exiting. 0 indicates no timeout.",
```

Comment:
Might be cool to make this a duration type and let users enter `1m` or 2h`, etc. Otherwise, you've got to do the math to figure out the number of seconds.

Reply:
nit: Maybe print the default to the help so users know what it is if not defined?

Reply:
cobra adds the default; this was copied from my console just now:
```
... before exiting. 0 indicates no timeout. (default 10800)
```